pcdsdevices/valve.py

```diff
@@ -226,6 +226,23 @@ class VGC(VRC):
     mps_state = Cpt(EpicsSignalRO, ':MPS_FAULT_OK_RBV', kind='omitted',
                     doc=('individual valve MPS state for debugging'))
 
+class VFS(VRC):
+    """Class for Fast Shutter Valve."""
+    vfs_closed = Cpt(EpicsSignalRO, ':CLS_RBV', kind='normal', 
+                     doc='Fast Shutter Closed Status')
+    vfs_open = Cpt(EpicsSignalRO, ':OPEN_RBV', kind='normal',
+                   doc='Fast Shutter Open Status')
+    vfs_vac_fault_ok = Cpt(EpicsSignalRO, ':FAULT_OK_RBV', kind='normal',
+                           doc='Fast Shutter Vacuum Fault OK')
+    vfs_vac_fault_reset = Cpt(EpicsSignalWithRBV, ':FAULT_RESET',
+                              kind='normal',
+                              doc='Fast Shutter Vacuum Fault Reset')
+    vfs_mps_ok = Cpt(EpicsSignalRO, ':FFO_OK_RBV', kind='normal',
+                     doc='Fast Shutter Fast Fault Output OK')
+    vfs_trigger = Cpt(EpicsSignalRO, ':TRIG_RBV', kind='normal',
+                      doc='Fast Sensor Input Trigger')
+    vfs_close_command = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal'
```

Comment:
Oops, missed this comma first time through
```suggestion
    vfs_close_command = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal',
```

Reply:
Thanks for catching this!